dir.props

```diff
@@ -546,6 +498,7 @@
     <SkipTests Condition="'$(SkipTests)'=='' and ('$(OsEnvironment)'=='Windows_NT' and '$(TargetsWindows)'!='true' and '$(OSGroup)'!='AnyOS')">true</SkipTests>
   </PropertyGroup>
 
-  <Import Project="$(RoslynPropsFile)" Condition="'$(OsEnvironment)'=='Unix' and Exists('$(RoslynPropsFile)')" />
-
+  <!-- Use Roslyn Compilers to build -->
+  <Import Project="$(RoslynPropsFile)" Condition="'$(OSEnvironment)'!='Unix' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false'" />
```

Comment:
The CentOS failure I was investigating, below, seems to have "disappeared".  I'm still attempting to get a repro though in case it's an intermittent failure I'm introducing.  Unless someone says that's a known issue, I plan to spend at least another day trying to get a repro before checking in. :/